rpcs3/rpcs3qt/kernel_explorer.cpp

```diff
@@ -152,13 +153,16 @@ kernel_explorer::kernel_explorer(QWidget* parent) : QDialog(parent)
 {
 	setWindowTitle(tr("Kernel Explorer"));
 	setObjectName("kernel_explorer");
-	setAttribute(Qt::WA_DeleteOnClose);
 	setMinimumSize(QSize(700, 450));
 
 	QVBoxLayout* vbox_panel = new QVBoxLayout();
 	QHBoxLayout* hbox_buttons = new QHBoxLayout();
 	QPushButton* button_refresh = new QPushButton(tr("Refresh"), this);
+	//QPushButton* button_log = new QPushButton(tr("Log"), this); // TODO
```

Comment:
what's with this unused code.
either implement it or don't commit it

Reply:
Fixed.